gradle.properties

```diff
@@ -7,3 +7,7 @@ repoid555=HDP-SOLR-2.6-100
 # Solr 6.6.2 repo
 # TODO a Solr repo is needed
 repoid662=HDP-SOLR-2.6-100
+
+# Solr 7.3.1 repo
+# TODO a Solr repo is needed
+repoid731=HDP-SOLR-2.6-100
```

Comment:
[Q] I blindly copied the existing repoid555 and repoid662 here.  As I understand these constants, they're used to specify the name of the artifact repo over in `src/main/template/repos/repoinfo.xml`.

But in that case, I'm surprised that both 5.5.5 and 6.6.2 have the same value.  I'd gathered other places that Solr 5.5.5 was mainly supported on HDP 2.4.  Maybe I'm conflating the HDP version and the HDP-search version though.  Just thought I'd ask in case I'm missing something here.

Relatedly, through LWSHADOOP-756, we can now package up rpm's/debs's that have Solr 7.3.1 on them.  But when I'm testing out this mpack, it's going to try and download/install a deb/rpm that HWX doesn't have hosted yet.  Do we ever send HWX test deb's/rpm's so that we can test out the mpack's "install" lifecycle phase?  I can always install the deb manually with `dpkg` I guess, but...